dhis-2/dhis-web-api/src/main/resources/org/hisp/dhis/webapi/servlet/login.html

```diff
@@ -321,9 +325,26 @@ <h1>Log in</h1>
     </div>
 
     <script>
+      // This function will dynamically determine the base URL of the current page
+      function getBaseUrl() {
```

Comment:
I added the `getBaseUrl ` function to dynamically get base url depending on the version, If you think there's a better way to do this from the server, I'm open to that conversation

Reply:
I have used a relative path instead of the `getBaseUrl ` function